openedx/tutor-contrib-oars
# feat: Adds Video events chart

Adds a Video events chart to the dashboard, displacing the generic xAPI Problem Events chart.

nit: I also renamed the `tutoroars/templates/app/data/assets` directory to `superset`, so it's clear where each of these OARS data directories are used.

**Screenshot**

<img width="463" alt="video_events" src="https://user-images.githubusercontent.com/7556571/226477188-fa978b01-fede-4c28-ab6c-c5ccf6d622fd.png">

**Testing instructions**

Since this only adds things to the Superset assets, if you have a running OARS stack, you can just:

1. Checkout this branch
2. Run `tutor config save`
3. Run `tutor dev do init -l oars`
4. View the Dashboard -- you should now see Video events chart that you can filter by course/block.
   **Tip**: Start typing "video" into the Dashboard's block filter field to select from the available video blocks.
